add navigation

desska opened the pull request:
add fragments:

fragment_search
fragment_vacancy
fragment_filter_settings
fragment_filter_location
fragment_filter_country
fragment_filter_region
fragment_filter_industry

desska committed:
add navigation

add fragments:

fragment_search
fragment_vacancy
fragment_filter_settings
fragment_filter_location
fragment_filter_country
fragment_filter_region
fragment_filter_industry

skilanet commented on app/src/main/res/navigation/navigation_graph.xml:
```diff
@@ -0,0 +1,72 @@
+        android:name="ru.practicum.android.diploma.ui.filtercountry.FilterCountryFragment"
+        android:label="fragment_filter_country"
+        tools:layout="@layout/fragment_filter_country" >
+        <action
```
мне кажется, что это лишнее действие

skilanet reviewed (commented):
Мне кажется что на фрагментах стоит отображать название фрагмента 
Для понимания навигации добавить на фрагментах кнопочки хотя бы просто какие-нибудь
И наверное стоит убрать //TODO коменты, чтобы на них detekt не ругался

desska committed:
add navigation

add fragments:

fragment_search
fragment_vacancy
fragment_filter_settings
fragment_filter_location
fragment_filter_country
fragment_filter_region
fragment_filter_industry

desska committed:
Merge branch 'develop' into Epic0_navigation
